src/nyc_trees/nyc_trees/context_processors.py

```diff
@@ -8,13 +8,23 @@
 from calendar import timegm
 from datetime import datetime
 
+
 from django.utils.timezone import make_aware, utc
 
+from apps.event.models import EventRegistration
 from apps.survey.models import (Blockface, Survey, Territory,
                                 BlockfaceReservation)
 from apps.users.models import TrustedMapper
 
 
+def my_events_now(request):
+    user = request.user
+    if user.is_authenticated():
+        events = EventRegistration.my_events_now(user)
+        if len(events) > 0:
```

Comment:
Consider always populating `my_events_now` on the context to avoid null/length checks on the templates. Not sure what the Django best practices are surrounding this.